dashboard/test/models/programming_expression_test.rb

```diff
@@ -53,8 +53,8 @@ class KeyConstraintTests < ActiveSupport::TestCase
 
   test "can serialize and seed programming expression" do
     programming_environment = create :programming_environment
-    exp = create :programming_expression, key: 'myExp', category: 'World', examples: 'myexamples', palette_params: 'some parameters', programming_environment_id: programming_environment.id
-    exp.color = ProgrammingExpression.get_category_color('World')
+    category = create :programming_environment_category, programming_environment: programming_environment, name: 'World', color: '#ABCDEF'
+    exp = create :programming_expression, key: 'myExp', category: 'World', examples: 'myexamples', palette_params: 'some parameters', programming_environment_id: programming_environment.id, programming_environment_category_id: category.id
```

Comment:
Shoudl this still have category set separate from programming_environment_category?

Reply:
Yeah probably. I'll add that back.

Reply:
Oh sorry, misunderstood the comment. I think that's fine for now as we still seed and serialize the category column